packages/seedless-onboarding-controller/src/SeedlessOnboardingController.test.ts

```diff
@@ -3747,6 +3762,13 @@ describe('SeedlessOnboardingController', () => {
         OLD_PASSWORD,
       );
 
+      initialSeedlessEncKey =
+        mockToprfEncryptor.deriveSeedlessEncKey(OLD_PASSWORD);
+
+      const aes = managedNonce(gcm)(mockResult.pwEncKey);
+      initialEncryptedSeedlessEncryptionKey = aes.encrypt(
+        initialSeedlessEncKey,
+      );
```

Comment:
this needs to be an encryption of an exported key, not a raw key.
see `encryptWithDetail`.